draft-ietf-quic-ack-frequency.md

```diff
@@ -365,6 +365,16 @@ reducing the ACK rate compared to {{Section 13.2.1 of QUIC-TRANSPORT}} during
 extreme congestion or when peers are using DCTCP {{?RFC8257}} or other
 congestion controllers that mark more frequently than classic ECN {{?RFC3168}}.
 
+If the most recent ACK_FREQUENCY frame an endpoint has received from the peer
+has an `Ignore CE` value of `true` (0x01), receipt of a CE marked packet
+SHOULD NOT cause an endpoint to send an immediate acknowledgement.  The endpoint
+still sends an immediate acknowledgement if it would have for a non CE marked
+packet.  Otherwise, the CE marks are reported in the next acknowledgement.
+
+The Ignore-CE bit SHOULD NOT be set if ECT(1) is negotiated with DCTCP
```

Comment:
Technically ECT(1) is not negotiated, the sender just sends it. It is validated though to check for network and receiver support. Suggestion for text,
```
The Ignore-CE bit SHOULD NOT be set if the sender sets ECT(1) in its outgoing packets which maybe used with DCTCP {{}} or L4S
``` 

One question, does DCTCP use ECT(1), I thought it was using ECT(0).

Reply:
Sorry, it appears you're correct about DCTCP.  I'm going to remove that.

Reply:
Removed DCTCP and accepted your suggestion with a minor change.